src/Components/Form/Form.js

```diff
@@ -149,3 +95,11 @@ export default class Form extends React.Component {
         );
     }
 }
+
+const defaults = {
+    programId: 4,
+    materialId: 1,
+    ionId: 1,
```

Comment:
```suggestion
    programId: 2, // PSTAR https://github.com/APTG/libdedx/blob/v1.2.1/libdedx/dedx_program_const.h#L8
    materialId: 276, // liquid WATER https://github.com/APTG/libdedx/blob/v1.2.1/libdedx/dedx_program_const.h#L197
    ionId: 1, // currently proton (HYDROGEN)  https://github.com/APTG/libdedx/blob/v1.2.1/libdedx/dedx_program_const.h#L244
```

Reply:
I've cross-checked the codes with https://aptg.github.io/libdedx/#a-3-additional-lists